components/nav/NavFooter.vue

```diff
@@ -9,6 +9,11 @@ function toggleDark() {
 
 <template>
   <footer p4 text-sm text-secondary-light flex="~ col">
+    <div mb-15px class="beta-label">
+      <b>Mozilla Social is an experimental beta product.</b> We would love <NuxtLink href="https://survey.alchemer.com/s3/7470063/mozilla-social-feedback-form" target="_blank" external>
+        <u>your feedback</u>
+      </NuxtLink>.
+    </div>
```

Comment:
We should sort out how to get this ready for localization.  Having the NuxtLink in there might make this ... interesting 😉

Reply:
Check out the `i18n-t` component; it's designed for situations like this: https://vue-i18n.intlify.dev/guide/advanced/component.html

I used it in my Report Modal PR, so you can look at that again as an example.

As Joel predicted, it is a little... interesting.

Reply:
@aaga was able to figure it out, thanks!! That wasn't _too_ bad, we did similar in the Pocket repo